### ISSUE TITLE
Add missing marketing_permissions parameter to comply GDPR

### ISSUE THREAD

**Issue by jobee** [bwaidelich/Wwwision.Neos.MailChimp#17]
Hey @bwaidelich 
MailChimp has added a new parameters to be comply with the GDPR.

Adding an extra paramter to the `subscribe` method should quickly fix that, at least for those who just use the MailChimpService class.

```php
public function subscribe($listId, $emailAddress, array $additionalFields = null, array $marketingPermissions = null)
    {
        $subscriberHash = md5(strtolower($emailAddress));
        $arguments = [
            'email_address' => $emailAddress,
            'status' => 'pending',
            'marketing_permissions' => $marketingPermissions
        ];
        if ($additionalFields !== null) {
            $arguments['merge_fields'] = $additionalFields;
        }
        $this->put("lists/$listId/members/$subscriberHash", $arguments);
    }
```
Do you prefer a PR for that?

See also `marketing_permissions` in https://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/

**Comment by bwaidelich** [comment]
@jobee Thanks for the heads up

> Do you prefer a PR for that?

Yes, please! Cheers!

**Comment by jobee** [comment]
Still searching for the required `marketing_permission_id` s in MailChimp backend to test this :-/ It seems they are not available/visible via MailChimps form builder... will provide a PR as soon as I figured this out

**Comment by jobee** [comment]
ok... `marketing_permission_id` s are currently only available via API request. The easiest way to get the correct IDs for configured GDPR options is probably via MailChimps API playground by just viewing a member of a list and searching for `marketing_permission_id`

**Comment by bwaidelich** [comment]
Solved with #18
